src/interface/thirdparty-pisp-api.yaml

```diff
@@ -632,3 +600,143 @@ paths:
+        - name: Content-Length
+          in: header
+          required: false
+          schema:
+            type: integer
+          description: |
+            The `Content-Length` header field indicates the anticipated size of the
+            payload body. Only sent if there is a body.
+            **Note:** The API supports a maximum size of 5242880 bytes (5 Megabytes).
+        - name: Content-Type
+          in: header
+          schema:
+            type: string
+          required: true
+          description: |
+            The `Content-Type` header indicates the specific version of the API used
+            to send the payload body.
+        - name: Date
+          in: header
+          schema:
+            type: string
+          required: true
+          description: The `Date` header field indicates the date when the request was sent.
+        - $ref: '#/paths/~1parties~1%7BType%7D~1%7BID%7D/get/parameters/2'
+        - $ref: '#/paths/~1parties~1%7BType%7D~1%7BID%7D/put/parameters/1'
+      requestBody:
+        required: true
+        content:
+          application/json:
+            schema:
+              title: ThirdpartyTransactionIDPatchResponse
+              type: object
+              description: 'The object sent in the PATCH /thirdpartyRequests/transactions/{ID} callback.'
+              properties:
+                fulfilment:
+                  title: IlpFulfilment
+                  type: string
+                  pattern: '^[A-Za-z0-9-_]{43}$'
+                  maxLength: 48
+                  description: Fulfilment that must be attached to the transfer by the Payee.
+                  example: WLctttbu2HvTsa1XWvUoGRcQozHsqeu9Ahl2JW9Bsu8
+                completedTimestamp:
+                  title: DateTime
+                  type: string
+                  pattern: '^(?:[1-9]\d{3}-(?:(?:0[1-9]|1[0-2])-(?:0[1-9]|1\d|2[0-8])|(?:0[13-9]|1[0-2])-(?:29|30)|(?:0[13578]|1[02])-31)|(?:[1-9]\d(?:0[48]|[2468][048]|[13579][26])|(?:[2468][048]|[13579][26])00)-02-29)T(?:[01]\d|2[0-3]):[0-5]\d:[0-5]\d(?:(\.\d{3}))(?:Z|[+-][01]\d:[0-5]\d)$'
+                  description: |
+                    The API data type DateTime is a JSON String in a lexical format that is
+                    restricted by a regular expression for interoperability reasons.
+                    The format is according to
+                    [ISO 8601](https://www.iso.org/iso-8601-date-and-time-format.html),
+                    expressed in a combined date, time and time zone format.
+                    A more readable version of the format is yyyy-MM-ddTHH:mm:ss.SSS[-HH:MM].
+                    Examples are \"2016-05-24T08:38:08.699-04:00\", \"2016-05-24T08:38:08.699Z\"
+                    (where Z indicates Zulu time zone, same as UTC).
+                  example: '2016-05-24T08:38:08.699-04:00'
+                transferState:
+                  title: TransferState
+                  type: string
+                  enum:
```

Comment:
While technically true, I think the only reasonable values a PISP would expect are `COMMITTED` or perhaps `RESERVED` - other responses would trigger a `PUT /thirdpartyRequests/transactions/{id}/error`.

But I'm happy to leave this here, since it makes it obvious that we are just referencing the `FSPIOP-API` spec

Reply:
I just realised I'm looking at the PISP repo, and not api-snippets. So just ignore that comment since it doesn't really belong here anyway.